capstone-app/src/app/modules/maps/maps.module.ts

```diff
@@ -2,12 +2,20 @@ import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { MapComponent } from './map/map.component';
 import { GoogleMapsModule } from '@angular/google-maps';
+import { MatIconModule } from '@angular/material/icon';
+import { MatButtonModule } from '@angular/material/button';
+import { MatInputModule } from '@angular/material/input';
+import { MatAutocompleteModule } from '@angular/material/autocomplete';
```

Comment:
The way we have our webapp structured is that we know we are all going to be using Angular Material so there's a shared module dedicated to material imports. All you need to do is move those imports there and then import the Material module here